src/recipe/recipe.controller.ts

```diff
@@ -48,6 +53,11 @@ export class RecipeController {
     const createdRecipe = await this.recipeService.createRecipe(
       createRecipeRequest,
     );
+    await this.redisCacheService.set(
+      `cache:recipe:${createdRecipe.id}`,
```

Comment:
Consider extracting a function for creating a key for a cached recipe.
It's super easy to do a typo in the keys (as they're usually strings) and super hard to find such an issue.

```ts
recipeCacheKey(recipeId: string): string {
  return `cache:recipe:${createdRecipe.id}`;
}
```
and use it like this
```suggestion
      recipeCacheKey(createdRecipe.id),
```